src/OrchardCore/OrchardCore.ResourceManagement/TagHelpers/StyleTagHelper.cs

```diff
@@ -1,14 +1,18 @@
 using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.AspNetCore.Razor.TagHelpers;
 
 namespace OrchardCore.ResourceManagement.TagHelpers
 {
     [HtmlTargetElement("style", Attributes = NameAttributeName)]
     [HtmlTargetElement("style", Attributes = SrcAttributeName)]
+    [HtmlTargetElement("style", Attributes = AtAttributeName)]
     public class StyleTagHelper : TagHelper
```

Comment:
I'm surprised we even support the `At` attribute for styles. I can't think why or how it could eve be anything but placed at the `Head`. Maybe just to be explicit?

A hint for when you get to looking at how the dependencies work

They require a `Name` attribute to work, so I would use that as your defining `HtmlTargetElement` rather than `At`

Reply:
IMHO add new StylesheetTagHelper for inline styles that supports both css/less/scss